rfc/rfc-0000-lightweight-rfc-process.md

```diff
@@ -0,0 +1,66 @@
+# RFC-0000: Lightweight RFC Process for riff
+
+**Authors:** Swapnil Bawaskar
+
+**Status:**
```

Comment:
```suggestion
**Status:** Accepted
```

Reply:
LGTM

@sbawaska feel free to merge after committing the status to Accepted as suggested here

Nice job, and thank you!

Reply:
+1